docs/kubernetes/windows_demo.md

```diff
@@ -0,0 +1,181 @@
+    git remote add demo https://github.com/almos98/kubernetes
+    ```
+2. Apply changes to personal fork.
+    ```bash
+    git fetch demo refs/heads/CSI-prototype
+    git checkout -b csi_prototype
+    COMMIT=$(git ls-remote demo | grep refs/heads/CSI-prototype | cut -f 1)
+    git cherry-pick $COMMIT
+    ```
+
+3. Build Windows binaries.
+    ```bash
+    curl https://github.com/yujuhong/kubernetes/commit/27e608a050a997be5ab736a7cdeb29aa68f3b7ee.patch | git apply
+    make clean
+    make quick-release
+    ```
+
+### Node Driver Registrar
+1. Add [Almos98/node-driver-register](https://github.com/almos98/node-driver-registrar) as a remote.
+    ```bash
+    git checkout master
+    git remote add demo https://github.com/almos98/node-driver-registrar
+    ```
+2. Apply changes to personal fork.
+    ```bash
+    git fetch demo refs/heads/CSI-Prototype
+    git checkout -b csi_prototype
+    COMMIT=$(git ls-remote demo | grep refs/heads/CSI-Prototype | cut -f 1)
+    git cherry-pick $COMMIT
+    git push # Needed for step 4
+    ```
+3. Build Windows binary and upload it to bucket.
+    ```bash
+    make
+    CLOUDSDK_CORE_PROJECT=<your project>
+    gsutil mb gs://${CLOUDSDK_CORE_PROJECT}-bucket
+    gsutil cp bin/csi-node-driver-registrar.exe gs://${CLOUDSDK_CORE_PROJECT}-bucket/
+    ```
+4. Build the Windows container image.
+    - Clone your personal fork on your non-cluster Windows VM.
+    - Copy the binary from your bucket.
+        ```Powershell
+        $PROJECT='<your project>'
+        cd node-driver-registrar
+        git checkout CSI-Prototype
+        gsutil cp gs://$PROJECT-bucket/csi-node-driver-registrar.exe bin/
+        ```
+    - Build and upload the image.
+        ```PowerShell
+        docker build -f Dockerfile.Windows . --tag csi-node-driver-registrar:1809
+        docker tag csi-nide-driver-registrar:1809 gcr.io/$PROJECT/csi-node-driver-registrar:1809
+        docker push gcr.io/$PROJECT/csi-node-driver-registrar:1809
+        ```
+### FileStore CSI Driver
+0. One-time per project: Create GCP service account for the CSI driver and set the Cloud Filestore editor role.
+    ```bash
+    ./deploy/project_setup.sh
+    ```
+1. Add [Almos98/gcp-filestore-csi-driver](https://github.com/almos98/gcp-filestore-csi-driver) as a remote.
+    ```bash
+    git checkout master
+    git remote add demo https://github.com/almos98/gcp-filestore-csi-driver
+    ```
+2. Apply changes to personal fork.
+    ```bash
+    git fetch demo refs/heads/windows_demo
+    git checkout -b csi_prototype
+    COMMIT=$(git ls-remote demo | grep refs/heads/windows_demo | cut -f 1)
+    git cherry-pick $COMMIT
+    ```
+3. Build Windows binary and upload it to the bucket.
+    ```bash
+    make windows-local
+    gsutil cp bin/gcfs-csi-driver.exe gs://$PROJECT-bucket/
+    ```
+## Preparing the cluster
+1. Go back to your k/k workspace where the patched kubernetes binaries were built and bring up a cluster using those binaries.
```

Comment:
You listed having a cluster as a prerequisite above, but that's not necessary because this step explains the cluster setup. You can remove the step in the prerequisites above, but add a link to https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/windows/README-GCE-Windows-kube-up.md here for further reference.